### DISCUSSION TITLE
Pinia Nuxt not working

### DISCUSSION THREAD

**Discussion by typhoon11** [vuejs/pinia#1643] [Help and Questions]
### Reproduction

Not working

### Steps to reproduce the bug

After installing @pinia/nuxt and starting the nuxt project it throws error  
Cannot start nuxt:  Cannot find module 'pinia/dist/pinia.mjs'   

### Expected behavior

It should work

### Actual behavior

It doesn't work

### Additional information

Nuxt version is 3.0.0-rc.9 and pinia nuxt version is 0.4.2

**Comment by a-toms** [comment]
I'm having this problem too on macos, Nuxt 3.0.0-rc.8. 

To replicate, simply create a new nuxt 3 project with nuxt/pinia listed as a dependency.

In the meantime, you can use composables in nuxt 3 to manage shared state: https://v3.nuxtjs.org/guide/features/state-management/

**Comment by TonyFresneau** [comment]
Hello, any news ?

**Comment by LastxTemplar** [comment]
I am getting the same error, seems like the `../dist/pinia.mjs` file is actually missing. Is there any solution for this, I can't actually use pinia at all because of this.

**Comment by mschadev** [comment]
## First Way
Add `alias.pinia` in `nuxt.config.ts`
```ts
import { createResolver } from "@nuxt/kit";
export default defineNuxtConfig({
  // ...
  modules: ["@pinia/nuxt"],
  alias: {
    pinia: createResolver(import.meta.url).resolve("node_modules/@pinia/nuxt/node_modules/pinia/dist/pinia.mjs"),
  },
  // ...
});
```
In this case, it works without installing `pinia`

## Second Way
```bash
npm install pinia # or npm install pinia --force
```

## Test Environment 
+ `nuxt`: `3.0.0-rc.11`
+  `@pinia/nuxt`: `0.4.2`
+ `pinia`: `2.0.22`

## Related Issues
https://github.com/nuxt/framework/issues/6623
